users/models.py

```diff
@@ -367,6 +367,10 @@ def get_all(cls):
             QuerySet: QuerySet of all users.
         """
         return cls.active_users.all()
+    
+    @property
+    def id(self):
+        return self.user_id
```

Comment:
Document in the model-level docstring or comments any reason for this aliasing, as it may be non-obvious for future maintainers.